catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/DashboardRepository.java

```diff
@@ -190,7 +197,14 @@ public EntityUpdater getUpdater(Dashboard original, Dashboard updated, boolean p
     }
     String dashboardId = dashboard.getId().toString();
     List<String> chartIds =
-        daoCollection.relationshipDAO().findTo(dashboardId, Entity.DASHBOARD, Relationship.HAS.ordinal(), Entity.CHART);
+        daoCollection
+            .relationshipDAO()
+            .findTo(
+                dashboardId,
+                Entity.DASHBOARD,
+                Relationship.HAS.ordinal(),
+                Entity.CHART,
+                toBoolean(Include.NON_DELETED));
```

Comment:
shouldn't this be based on dashboard.deleted status instead of passing NON_DELETED

Reply:
Yes, I'm fixing it.

Reply:
this should've failed the tests though. Can you double check if the tests going to catch these

Reply:
I'm on it.